doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -73,6 +73,80 @@ See :ref:`rocq_makefile` and :ref:`building_dune`.
       .. rocqdoc::
          -R <PATH> Mod1
 
+System configuration
+--------------------
+
+Running `rocq` (or the `coq` compatibility commands) will fail if it
+cannot find certain expected files (except for subcommands like
+`rocq wc` which do not need to find anything).
+
+The files are searched according to Rocq's build time configuration,
+the location of the `rocq` executable and the available command line
+arguments and environment variables.
+
+.. note::
+
+   If `configure` is not explicitly called, it is equivalent to
+   `configure -relocatable`.
+
+Let `$root` be the parent directory of the directory of the `rocq`
+executable (typically `rocq` is `$root/bin/rocq`, or
+`$root\bin\rocq.exe` on Windows).
+
+Let `$libdirconf` be the value passed to `configure -libdir`, or
+`coq/lib` if `libdir` was not used.
+
+Let `$libdir` be
+
+- if the `coqlib` command line argument was used, its value
+
+- otherwise, if the `ROCQLIB` environment variable is defined, its value
+
+- otherwise, if the deprecated `COQLIB` environment variable is defined, its value
+
+- otherwise, if `$libdirconf` is absolute, its value
+
+- otherwise, if `$root/$libdirconf/theories/Init/Prelude.vo` exists or
+  Rocq was configured with `-relocatable`, `$root/$libdirconf`
+
+- otherwise, if Rocq was configured with `-prefix $prefix`, `$prefix/$libdirconf`
+  (if the user gave a relative `$prefix`, it is turned into an absolute
+  path based on the working directory of the `configure` invocation)
+
```

Comment:
Maybe worth saying something like "unless -boot, Rocq acts as though the command line arguments included `-R $libdir/theories Corelib -Q $libdir/user-contrib ""`"

Reply:
I'd be great to actually specify / document that indeed.

Reply:
I did add it below (line 130)